src/gatling/simulations/uk/gov/hmcts/reform/exui/performance/scenarios/EXUIMCLogin.scala

```diff
@@ -175,11 +175,12 @@ object EXUIMCLogin {
         .exec(http("XUI${service}_020_025_GetWorkBasketInputs")
               .get("/data/internal/case-types/FinancialRemedyMVP2/work-basket-inputs")
               .headers(LoginHeader.headers_17))
-       /* .exec(getCookieValue(CookieKey("XSRF-TOKEN").withDomain("manage-case.perftest.platform.hmcts.net").saveAs("XSRFToken")))*/
-       /* .exec( session => {
+        .exec(getCookieValue(CookieKey("__auth__").withDomain("manage-case.perftest.platform.hmcts.net").saveAs("authToken")))
+        .exec(getCookieValue(CookieKey("XSRF-TOKEN").withDomain("manage-case.perftest.platform.hmcts.net").saveAs("XSRFToken")))
+        .exec( session => {
```

Comment:
Can you remove/comment this please, we don't need it for perf testing

Reply:
Done

Reply:
@amoghnaik-hmcts I still see it on line 180??